packages/cross-chain/src/interfaces/index.ts

```diff
@@ -1,8 +1,6 @@
 export * from "./crossChainProvider.interface.js";
-export * from "./actions/index.js";
-export * from "./AcrossProvider.interface.js";
-export * from "./SampleProvider.interface.js";
-export * from "./paramsParser.interface.js";
-export * from "./quoteAggregator.interface.js";
+export * from "./acrossProvider.interface.js";
```

Comment:
```suggestion
export * from "./AcrossProvider.interface.js";
```

I believe this is breaking the vercel deployment

Reply:
https://github.com/defi-wonderland/interop-sdk/pull/67/commits/f1c24786396d8bd497e07b5b3464c0462794d110